test/evp_extra_test.c

```diff
@@ -476,6 +477,97 @@ static int test_EVP_PKCS82PKEY(void)
 }
 #endif
 
+struct keys_st {
+    int type;
+    char *priv;
+    char *pub;
+} keys[] = {
+    {
+        EVP_PKEY_HMAC,
+        "0123456789",
+        NULL
+    }, {
```

Comment:
This indentation is kinda wonky.  I think most entries fit in one line, except those with big keys.